lib/ansible/module_utils/basic.py

```diff
@@ -2223,7 +2238,10 @@ def append_to_file(self, filename, str):
         fh.write(str)
         fh.close()
 
-    def pretty_bytes(self,size):
+    def bytes_to_human(self, size):
+        return self.pretty_bytes(size)
+
+    def pretty_bytes(self, size):
```

Comment:
Instead of this, I'd rename the function to the name we like and use assignment for the backwards-compat name.  Liek this:

def bytes_to_human(self, size):
    ranges = (
                   (1<<70, 'ZB'),
    [...]
    return '%.2f %s' % (float(size)/ limit, suffix)

pretty_bytes = bytes_to_human